crates/apub/src/activities/mod.rs

```diff
@@ -324,6 +346,6 @@ pub async fn match_outgoing_activities(
       }
     }
   };
-  spawn_try_task(fed_task);
+  fed_task.await?;
```

Comment:
i did try but would cause a fair amount of changes because it changes the behaviour of `?` operator  so it was a lot of changes i didn't want to do to make the PR noisier